src/app/account/account.service.ts

```diff
@@ -85,4 +85,9 @@ export class AccountService {
       headers: this.headers,
     }, {responseType: 'text'});
   }
+
+  updatePassword(passwordData: any): Observable<any> {
```

Comment:
I don't know if it's ok to have any here. We know what kind of data will go into this method. I think you should replace any with that interface you defined.